ci-operator/config/openshift/cluster-api/openshift-cluster-api-master.yaml

```diff
@@ -0,0 +1,43 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+binary_build_commands: make manager clusterctl
+canonical_go_repository: sigs.k8s.io/cluster-api
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: cluster-api
+resources:
+  '*':
+    limits:
+      cpu: '2'
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+tests:
+- as: unit
+  commands: make test
```

Comment:
how is this expected to work? since the tests require etcd/api-server/kubectl binaries https://github.com/kubernetes-sigs/cluster-api/blob/master/pkg/controller/machineset/machineset_controller_suite_test.go#L42. That's why https://github.com/kubernetes-sigs/cluster-api/blob/master/scripts/ci-test.sh#L24

Reply:
this should be solved by https://github.com/openshift/release/pull/2697/files#diff-f0e4e08ecb398ef3dfe3a832e3714cd7R33 now